### PR TITLE
Calendarsite

### LINKED ISSUE #21: As a user I want to edit my Day-Plan
- [] Add Calendar Page
- [] Map all Campsites at Calendar Page
- [] Add Buttons to edit the Day

### PR DESCRIPTION
**Pull request by m-wiemers**
Fix #21

### CODE REVIEW

File: pages/[day].tsx [hunk 1]
```diff
@@ -1,8 +1,48 @@
...
+      alert("No campsites for this day found...");
+      router.push("/map");
+    } else {
+      setDays(days);
+    }
+  }, []);
+
+  function handleDelete(e) {
+    e.preventDefault();
+    console.log("button clicked");
```
Comment (mewiemers) [inline pages/[day].tsx]: do you need this anymore?

**Review by mewiemers — APPROVED** [review]
LGTM!🦘

**Comment by colefriedlaender** [comment]
Good job!

**Review by DNaunin — APPROVED** [review]
LGTM! 🦘

**Pull request merged**